docs/tutorials/ssr/README.md

```diff
@@ -1,84 +1,135 @@
 # Firebase Auth in SSR/Universal Mode
 
 <Badge text="EXPERIMENTAL" type="warn"/>
+
 This module provides an option for the easy setup of **server-side authentication** as described in [this article](https://firebase.google.com/docs/auth/web/service-worker-sessions) of the official Documentation.
 
-### Step 0 - Install Dependencies
+## Step 0 - Install Dependencies
 
-In addition to @nuxtjs/firebase, install these two dependencies:
+Install `firebase-admin` and `@nuxtjs/pwa`:
 
 ```bash
-yarn add firebase-admin # OR npm i firebase-admin
-yarn add jwt-decode # OR npm i jwt-decode
+yarn add firebase-admin @nuxtjs/pwa # OR npm i firebase-admin @nuxtjs/pwa
```

Comment:
can we add this change to do **upgrade-guide**, too?

Reply:
Done (please check)